10up/ElasticPress
# WP CLI instant results template commands

### Description of the Change



This PR implements #3150


### How to test the Change


Test the three new commands:

```
wp elasticpress get-search-template <--- Should retrieve the search template in JSON
wp elasticpress put-search-template <--- Should send the search template
wp elasticpress delete-search-template <--- Should delete search template
```

### Changelog Entry

> Added - WP CLI commands  get-search-template put-search-template delete-search-template 
> Changed - Existing functionality
> Deprecated - Soon-to-be removed feature
> Removed - Feature
> Fixed - Bug fix
> Security - Vulnerability


### Credits

Props @oscarssanchez 


### Checklist:
